docs/nested.md

```diff
@@ -1 +1,63 @@
 ## Nested Selectors
+
+Sometimes you will want to nest selectors to target only elements inside the current class or React component. Here is an example of a simple element selector nested in the class generated with `css`:
+
+```jsx
+import { css } from 'emotion';
+
+const paragraph = css`
+  color: gray;
+
+  & a {
+    border-bottom: 1px solid currentColor;
+  }
+`;
+```
+
+For now, the `&` before the selector is required. You can also select the current class nested in another element:
+
+```jsx
+const paragraph = css`
+  color: gray;
+
+  header & {
+    color: black;
+  }
+`;
+```
+
+To nest a class selector using the class generated with `css`, simply interpolate it:
+
+```jsx
+const link = css`
+  color: hotpink;
+`;
+
+const paragraph = css`
+  color: gray;
+
+  & ${link} {
```

Comment:
Does this really work? I think it would but you would need the `.` before the interpolation.